packages/bson-transpilers/codegeneration/DeclarationStore.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Stores declarations for the driver syntax
+ *
+ * @returns {object}
+ */
+class DeclarationStore {
+  constructor() {
+    this.store = [];
+  }
+
+  /**
+   * Add declaration statements by a pre-incremented variable root-name
+   *
+   * @param {string} varRoot - The root of the variable name to be appended by the occurance count
+   * @param {function} declaration - The code block to be prepended to the driver syntax
+   * @returns {string} the variable name with root and appended count
+   */
+  add(varRoot, declaration) {
+    const varName = this.next(varRoot);
+    const data = {varName: varName, declaration: declaration(varName) };
```

Comment:
```suggestion
    const data = { varName: varName, declaration: declaration(varName) };
```

(surprised the linter didn’t say anything)